motor_vehicle_report_test.go

```diff
@@ -0,0 +1,23 @@
+package checkr
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestClient_GetMotorVehicleReport(t *testing.T) {
+	candidate := createCandidate(t)
+	reqPayload := &CreateReportRequest{
+		CandidateID: candidate.ID,
+		Package:     driverpackage,
+	}
+	client := newClient(t)
+	rpt, err := client.CreateReport(reqPayload)
+	require.NoError(t, err)
+	require.NotNil(t, rpt)
+	ssnTrace, err := rpt.GetMotorVehicleReportSearch(rpt.MotorVehicleReportID, client)
```

Comment:
Be sure to also change the variable names when you're copy-pasting - `ssnTrace`.